packages/db/src/mock-data/friend.json

```diff
@@ -1,42 +1,2 @@
 [
```

Comment:
I deleted this because it uses a relationship and we can't set those in the mock data i think

Reply:
Oh I SEEE

Reply:
Yeah but I'm not sure because sometimes they're set whenever we seed so maybe we could look into it

Reply:
hmm okay, yea it would be nice to have the relationships with the seeds

Reply:
ye we could do that, issue opened